internal/hcsoci/resources.go

```diff
@@ -8,73 +8,106 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// NetNS returns the network namespace for the container
+func (r *Resources) NetNS() string {
+	return r.netNS
+}
+
+// Resources is the structure returned as part of creating a container. It holds
+// nothing useful to clients, hence everything is lowercased. A client would use
+// it in a call to ReleaseResource to ensure everything is cleaned up when a
+// container exits.
 type Resources struct {
```

Comment:
Probably better to have Resources be an interface and each resource type be a struct of its own kind with its own release impl's right?

Reply:
Yeah, this would be a follow-up. John did the refactoring to Resources originally.